src/object/srv_obj_migrate.c

```diff
@@ -1241,7 +1241,12 @@ migrate_fetch_update_bulk(struct migrate_one *mrone, daos_handle_t oh,
 	/* For EC object, if the migration include both extent from parity rebuild
 	 * and extent from replicate rebuild, let rebuild the extent with parity first,
 	 * then extent from replication.
+	 *
+	 * Since the parity shard epoch should be higher or equal to the data shard epoch,
```

Comment:
just confirm that for example firstly partial write [0-1M]@epoch_10, then full stripe_write [0-4M]@epoch_20.
when rebuild shard 0, the [0-1M]@epoch_10 will be hidden and can not be iterated out and then need not rebuild?

So we only need to consider the case of data ext epoch >= parity ext epoch?